### PR TITLE
IDEA-224584: Fix filter

### PR DESCRIPTION
**Pull request by heckad**
https://youtrack.jetbrains.com/issue/IDEA-224584

### CODE REVIEW

**Comment by ybedrov** [comment]
Hello!

Have you submitted a contributor license agreement? If not, please follow the steps described at http://www.jetbrains.com/agreements/cla/ to sign it.

**Comment by heckad** [comment]
@ybedrov, yes I have.

File: platform/lang-impl/src/com/intellij/ide/util/PsiElementListCellRenderer.java [hunk 1]
```diff
@@ -321,7 +321,7 @@ public void installSpeedSearch(PopupChooserBuilder<?> builder, boolean includeCo
   }
 
   public void installSpeedSearch(IPopupChooserBuilder builder) {
-    installSpeedSearch(builder, false);
```
Comment (yole) [inline platform/lang-impl/src/com/intellij/ide/util/PsiElementListCellRenderer.java:324]: Please don't change the default behavior of this method. If you need `includeContainerText` to be true in a specific place, please make a change to that place.

**Review by yole — CHANGES_REQUESTED** [review]

**Comment by yole** [comment]
Merged manually, thanks!

**Pull request closed without merging**